cmd/web/web.go

```diff
@@ -80,6 +103,24 @@ func serve(cmd *cobra.Command, args []string) {
 		cancel()
 	}()
 
+	app := initApp()
+	if err := app.Start(ctx); err != nil {
+		log.Fatal("Error while running the web application server: ", err)
+	}
+}
+
+func prune(cmd *cobra.Command, args []string) {
+	olderThan := viper.GetDuration("prune-events")
+
+	app := initApp()
```

Comment:
I'm not sure if we should have a common `initApp` for this, as the variables checked in the `loadConfig` function don't even belong to this subcommand execution. In fact, this pruning shouldn't need any gin-gonic initialization code, which we need, as we create the app with these dependencies.
Anyway, this I guess is only if we want to be 100% purist.

Reply:
Let's follow up, we need to explore a bit the initialization in general

Reply:
Couldn't we instantiate only the service here? I don't see the reason why pruning events a method of `web.App`, when we only need it in a CLI command.

Reply:
@stefanotorresi thanks, i agree.
I will work on moving away the db initialization in the internal package so we can depend on it on both app and the command.

I also think we should move this out from the service, because the service also needs a channel to be initialized, something like an utility function in the datapipeline package will be enough.

Reply:
@stefanotorresi @arbulu89 refactored the db initialization and removed the prune method from the app